docs/pages/installation/generate_kubeconfig_eks.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
```

Comment:
Is there anything EKS-specific in here or is this the same script verbatim for GKE?

Reply:
I suppose to change to -d, but I didn't change. will change it again and do PR.